docs/resources/devurl.md

```diff
@@ -0,0 +1,63 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "coder_devurl Resource - terraform-provider-coder"
+subcategory: ""
+description: |-
+  Use this resource to define shortcuts to access applications in a workspace.
+---
+
+# coder_devurl (Resource)
+
+Use this resource to define shortcuts to access applications in a workspace.
+
+## Example Usage
+
+```terraform
+resource "coder_agent" "dev" {
+  os             = "linux"
+  arch           = "amd64"
+  dir            = "/workspace"
+  startup_script = <<EOF
+curl -fsSL https://code-server.dev/install.sh | sh
+code-server --auth none --port 13337
+EOF
+}
+
+resource "coder_devurl" "code-server" {
+  agent_id = coder_agent.dev.id
+  name     = "VS Code"
+  icon     = "builtin:vscode"
+  target   = "http://localhost:13337"
+}
+
+resource "coder_devurl" "vim" {
+  agent_id = coder_agent.dev.id
+  name     = "Vim"
+  icon     = "builtin:vim"
+  command  = "vim"
+}
```

Comment:
_DevURL_ is a bad name for this, which is why I'm open to change.

In a web terminal, this would open the command in that. In the CLI, this would open the command in the current terminal. You could open it with `coder open vim`.

Reply:
Ah, makes sense. I wasn't entirely caught up with that!

Reply:
Any ideas on a different name than DevURL?

Reply:
`app` is the only one that comes to mind, it is practical, but not great

Reply:
```tf
resource "coder_app" "vim" {
  agent_id = ...
  name = "Vim"
  icon = "builtin:vim"
  command = "vim"
}
```

Reply:
Yeah I think `coder_app` is better than `coder_devurl`. We could open to others for thoughts or go with this, your call

Reply:
```
provider.coder.url + "/icons/goland.png"
```